Torantulino/Auto-GPT
# Created autogpt module (renaming of scripts folder)

### Background
This PR addresses the need to reorganize our codebase for better maintainability and modularity. We have decided to rename the `scripts` folder to `autogpt`, as it will house the new `autogpt` module. This change lays the groundwork for future modularization efforts, allowing us to build upon this foundation as the project grows and evolves.

### Changes
- Rename the `scripts` folder to `autogpt`.
- Update all import statements and references to the newly renamed `autogpt` module.
- Ensure that existing functionality remains intact during the transition.

### Documentation
- Updated README file, to reflect the new module name.

### Test Plan
1. Manually test the application with various prompts to ensure that existing functionality is preserved.
2. Run existing test cases to verify that the changes do not introduce any regressions.
3. Update test cases to use the new `autogpt` module name, if necessary.

### PR Quality Checklist
